C/ds/FindLinkedListPalindrome.c

```diff
@@ -0,0 +1,154 @@
+/*C program to find whether a linked list is palindrome or not
+Using two pointers approach i.e slow(moves at x speed)pointer and fast(moves at
+2x speed)pointer By storing first half of the linked list in an array and then
+comparing each element from the other half of the linked list
+with the elements of array in reverse order till it get exhausted.
+
+Note:when no of elements are odd in Linked List skip the middle element
+Ex- Linked_List={1,2,3,4,3,2,1},size=7
+        Storing the first half in an array
+        arr=[1,2,3]
+
+        Linked_List[4]==arr[2]
+        Linked_List[5]==arr[1]
+        ...
+        ...
+        Palindrome(If all elements from array are matched)
+
+*/
+
+#include <malloc.h>
```

Comment:
Use `#include<bits/stdc++.h>` instead of `<malloc.h>`

Reply:
@ankitaggarwal23  I could have used <bits/stdc++.h> but only compiler like vscode supports that..unlike Devc++,Turbo C doesn't have any such Standard library. Even though u want me to change,I can do that  I don't have any issue.

Reply:
Please suggest should I need to change since my compiler(DevC++) is also not supporting.

Reply:
Please make the change. We will try to follow a standard